source/includes/linux/common-installation.rst

```diff
@@ -296,6 +296,11 @@ MinIO may log an increased number of non-critical warnings while the
 server processes connect and synchronize. These warnings are typically 
 transient and should resolve as the deployment comes online.
 
+.. versionchanged:: RELEASE.2023-02-09T05-16-53Z
+
+   Some drives may be offline when you start MinIO.
+   Enough drives must be online at start to achieve :ref:`write quorum <minio-ec-parity>` for the deployment.
```

Comment:
I think we could possibly merge these two:

```suggestion
MinIO starts once it detects enough drives to meet the :ref:`write quorum <minio-ec-parity>` for the deployment.
If any drives remain offline after starting MinIO, check and cure any issues blocking their functionality before starting production workloads.
```

Reply:
Your 301 suggestion implies that MinIO waits around until there are enough drives. THat's not it. It errors out.
Swapping `once` for `if` and otherwise taking your suggested edits.

Reply:
Ah yes wording sometimes i am good at it and sometimes i am not.

Thanks :D